__tests__/__snapshots__/CustomMenu.test.jsx.snap

```diff
@@ -0,0 +1,125 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Why are snaps appearing and yet they are in the ignore file?

Reply:
pushed this bit before adding it to the gitignore

Reply:
Kindly remove the `__tests__/` from this PR @RayNjeri